modules/ts-capi-migrate-unexpected-machine-counts-scaling.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ts-capi-migrate-unexpected-machine-counts-scaling_{context}"]
+= Unexpected machine counts after scaling
+
+On clusters that support migrating resources between the Machine API and the Cluster API, users might experience unexpected behavior when scaling the number of compute machines.
+The output of the `oc get` command for a compute machine set that does not use the authoritative API might contain inaccurate values in the `CURRENT`, `READY`, and `AVAILABLE` columns. 
+
+Cause::
+
+The values that populate the `CURRENT`, `READY`, and `AVAILABLE` columns originate in the `.status` stanza of a compute machine set.
+The two-way synchronization controller that handles resource conversion between authoritative API types does not synchronize values in the `.status` stanza.
++
+The value in the `DESIRED` column in the output of the `oc get` command for a compute machine set reflects the `.spec.replicas` value of a compute machine set.
+The two-way synchronization controller synchronizes values in the `.spec` stanza.
+
+Consequence::
+
+Users can expect to see the following behavior when scaling migrated machine sets:
++
+--
+. Start with a compute machine set with existing machines.
+. Migrate the machine set to use a different authoritative API.
+. Scale the migrated machine set up by setting a larger value in the `.spec.replicas` field.
+. The machine set creates machines with the current authoritative API to satisfy the number of requested replicas.
+. Scale the machine set down such that one of the following conditions causes the deletion of machines that do not use the current authoritative API:
+** Total number of replicas requested is fewer than the number of machines that do not use the current authoritative API.
+** The machine deletion policy for the machine set selects machines that do not use the current authoritative API.
+. Check the status of the nonauthoritative compute machine set by running the `oc get` command.
+** The value in the `DESIRED` column in the output reflects the `.spec.replicas` value.
+** The values in the `CURRENT`, `READY`, and `AVAILABLE` columns reflect the original number of replicas that existed before scaling the machine set.
+--
+
+Workaround::
+
+To verify that the scale-down operation successfully deleted the compute machines, run the `oc get` command that lists the compute machines.
```

Comment:
I think this might be a little confusing? We should still have the correct status on the authoritative machineset?

Reply:
Yep, this makes sense & looks good to me! Thanks! :)